noisyatom/config/urls.py

```diff
@@ -1,18 +1,7 @@
-"""noisyatom URL Configuration
```

Comment:
Why did you remove the doc string from the urls.py?

Reply:
We don't need this comments any more we already done what is saying in the doc string.